public/app/plugins/panel/timeseries/config.ts

```diff
@@ -181,6 +182,29 @@ export function getGraphFieldConfig(cfg: GraphFieldConfig): SetFieldConfigOption
         });
 
       commonOptionsBuilder.addStackingConfig(builder, cfg.stacking, categoryStyles);
+
+      builder.addSelect({
+        category: categoryStyles,
+        name: 'Transform',
```

Comment:
Not sure this name is clear enough, but can't think of anything else. Any suggestions are highly appreciated